tests/keras/layers/recurrent_test.py

```diff
@@ -232,6 +236,10 @@ def test_trainability(layer_class):
     assert len(layer.non_trainable_weights) == 0
 
 
+@pytest.mark.skipif(K.backend() == 'mxnet',
+                    reason='MXNet backend has an issue with Masking layer'
+                           'Tracking with this issue:'
+                           'https://github.com/awslabs/keras-apache-mxnet/issues/228')
```

Comment:
can you enable this test for unroll=True?
you can do the check in the test and don't execute the part that unroll=False

Reply:
Done. Can you please check it again?

Reply:
The `test_masking_correctness()` runs only with `unroll=False` so skipping the whole test. However `test_masking_layer()` runs with `unroll=True` and `unroll=False`, so skipped the `unroll=False` test.